### ISSUE TITLE
error when using the magic !anything

### ISSUE THREAD

**Issue by cetanu** [taverntesting/tavern#69]
yaml.constructor.ConstructorError: 
        could not determine a constructor for the tag 'tag:yaml.org,2002:python/object:builtins.object'

Running Tavern 0.7.1
PyYAML 3.12

Trying to match some things which should be in the response, but they are totally random

The test code to replicate:

    test_name: '#69'

    stages:
      - name: test the error
        request:
          url: https://jsonplaceholder.typicode.com/posts?title=qui%20est%20esse
          method: GET
        response:
          status_code: 200
          body:
            - userId: 1
              id: 2
              title: qui est esse
              body: !anything  

Any ideas?

**Comment by michaelboulton** [comment]
The code you've posted is missing the required "test_name" and "stages" for it to work:

```yaml
test_name: "abc"

stages:
  - name: test the error
    request:
      url: https://jsonplaceholder.typicode.com/posts?title=qui%20est%20esse
      method: GET
    response:
      status_code: 200
      body:
        - userId: 1
          id: 2
          title: qui est esse
          body: !anything
```
The error it's giving is a bit cryptic though, I'll look at that now

**Comment by michaelboulton** [comment]
This should be a bit better in 0.7.3 whenever the CI deploys it

**Comment by samiam109** [comment]
I've been getting an issue with magic variables !int and !anything. Example code with !int 
`test_name: test ledger post a debit for a user
includes:
  - !include ledger_includes.yaml

stages:
  - name: login with user to get auth token
    request:
      url: "{domain:s}/{user_login_path:s}"
      method: GET
      headers:
        Authorization: "{authentication_str:s}"
    response:
      status_code: 200
      save:
        body:
          auth_token: token

  - name: post a debit for a user using valid token then validate response
    request:
      url: "{domain:s}/{path_user_debit:s}"
      method: POST
      headers:
        Authorization: "{auth_token}"
      json: &param_schema
        address: "{debit_address:s}"
        amount: !int "{debit_amount:d}"
        currency: "{ledger_currency:s}"
        guid: "{debit_guid:s}"
        network: "{ledger_network:s}"
        reference: ""
    response:
      status_code: 200
      body: &response_schema
        $ext:
          function: tavern.testutils.helpers:validate_pykwalify
          extra_kwargs:
            schema:
              type: map
              mapping:
                message:
                  type: map
                  mapping:
                    id:
                      type: int
                      required: true
                    date_time_created:
                      type: str
                      required: true
                    last_updated:
                      type: str
                      required: true
                    sequence:
                      type: int
                      required: true
`
error:
`yaml.constructor.ConstructorError:` could not determine a constructor for the tag 'tag:yaml.org,2002:python/object:tavern.util.loader.TypeConvertToken'
  in "/tmp/tmpxmp4l9em.yaml", line 41, column 15`
When I try to use !anything
`test_name: test ledger get a user's address
includes:
  - !include ledger_includes.yaml

stages:
  - name: login with user to get auth token
    request:
      url: "{domain:s}/{user_login_path:s}"
      method: GET
      headers:
        Authorization: "{authentication_str:s}"
    response:
      status_code: 200
      save:
        body:
          auth_token: token

  - name: post a new user address using valid currency, network and auth token and validate response body
    request:
      url: "{domain:s}/{path_user_address:s}"
      method: POST
      headers:
        Authorization: "{auth_token}"
      json: &param_schema
        currency: "{ledger_currency:s}"
        network: "{ledger_network:s}"

    response:
      status_code: 200
      body: &response_schema
        $ext:
          function: tavern.testutils.helpers:validate_pykwalify
          extra_kwargs:
            schema:
              type: map
              mapping:
                message:
                  type: map
                  required: true
                  mapping:
                    address:
                      type: str
                      required: true
                    created:
                      type: str
                      required: true
                    currency:
                      type: str
                      required: true
                    network:
                      type: str
                      required: true
                    userid:
                      type: str
                      required: true

  - name: post a new user address using valid network and auth token and invalid currency. Check for error
    request:
      url: "{domain:s}/{path_user_address:s}"
      method: POST
      headers:
        Authorization: "{auth_token}"
      json:
        currency: "{invalid_currency:s}"
        <<: *param_schema
    response:
      status_code: 400
      body:
        error: !anything`
error:
`yaml.constructor.ConstructorError: could not determine a constructor for the tag 'tag:yaml.org,2002:python/object:builtins.object'
  in "/tmp/tmpjdvilk45.yaml", line 85, column 14
`

**Comment by michaelboulton** [comment]
This should be fixed in 0.8.0, There are also some new magic variables for matching specific types like `!anyint` and `!anyfloat` but I don't have time to update the documentation at the moment
